integration-tests/deployment/ccip/view/offramp.go

```diff
@@ -0,0 +1,80 @@
+package view
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/offramp"
+)
+
+type OffRamp struct {
+	Contract
+	DynamicConfig             offramp.OffRampDynamicConfig                `json:"dynamicConfig"`
+	LatestPriceSequenceNumber uint64                                      `json:"latestPriceSequenceNumber"`
+	SourceChainConfigs        map[uint64]offramp.OffRampSourceChainConfig `json:"sourceChainConfigs"`
+	StaticConfig              offramp.OffRampStaticConfig                 `json:"staticConfig"`
+	Owner                     common.Address                              `json:"owner"`
+}
+
+type OffRampReader interface {
+	TypeAndVersion(opts *bind.CallOpts) (string, error)
+	Address() common.Address
+	GetDynamicConfig(opts *bind.CallOpts) (offramp.OffRampDynamicConfig, error)
+	GetLatestPriceSequenceNumber(opts *bind.CallOpts) (uint64, error)
+	GetSourceChainConfig(opts *bind.CallOpts, sourceChainSelector uint64) (offramp.OffRampSourceChainConfig, error)
+	GetStaticConfig(opts *bind.CallOpts) (offramp.OffRampStaticConfig, error)
```

Comment:
This will create compatibility issue when offRamp gets updated to new version and this will get generated with a different gethwrapper package.For Example -[1_2_0](https://github.com/smartcontractkit/chainlink/tree/develop/core/gethwrappers/ccip/generated/evm_2_evm_offramp_1_2_0)  and [1_0_0](https://github.com/smartcontractkit/chainlink/tree/develop/core/gethwrappers/ccip/generated/evm_2_evm_offramp_1_0_0) , I have created contractwrapper package for this , eg - this [RMN method](https://github.com/smartcontractkit/chainlink/pull/14391/files#diff-6d727353c6806267c096478923a7ddb3c76299d3513ece3a769449af88c138dfR14) 
Please suggest if you can think of a better approach